pytradfri/aiocoap_api.py

```diff
@@ -0,0 +1,88 @@
+"""Coap implementation using aiocoap."""
+import json
+import logging
+
+import asyncio
+from aiocoap import Message, Context, GET, PUT
+from aiocoap.transports.tinydtls import PSK_STORE
+from pytradfri.command import Command
+from .parser import process_output
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def api_factory(host, security_code):
+    """Generate a request method."""
+
+    PSK_STORE[b'Client_identity'] = security_code.encode('utf-8')
```

Comment:
I think that for now we can just monkey patch the store. It's not clear to me how else it can be used.

We should probably ask the author.